crates/nu-cli/tests/completions/mod.rs

```diff
@@ -359,6 +359,30 @@ fn dotnu_completions() {
     assert!(!suggestions.is_empty());
 }
 
+#[test]
+fn dotnu_completions_const_nu_lib_dirs() {
+    let (_, _, engine, stack) = new_dotnu_engine();
+    let mut completer = NuCompleter::new(Arc::new(engine), Arc::new(stack));
+
+    // file in `lib-dir1/`
+    let completion_str = "use xyzz".to_string();
+    let suggestions = completer.complete(&completion_str, completion_str.len());
+    match_suggestions(&vec!["xyzzy.nu".into()], &suggestions);
+
+    // file in `lib-dir2/`
+    let completion_str = "use asdf".to_string();
+    let suggestions = completer.complete(&completion_str, completion_str.len());
+    match_suggestions(&vec!["asdf.nu".into()], &suggestions);
+
+    // if `./` specified by user, file in `lib-dir*` should be ignored
+    #[cfg(windows)]
+    let completion_str = "use .\\asdf".to_string();
+    #[cfg(not(windows))]
+    let completion_str = "use ./asdf".to_string();
```

Comment:
This should also work on Windows, forward slashes are supposed to be valid there.